### PR TITLE
aws cognitive polly

### PR DESCRIPTION
**Pull request by katarzyna-koltun-mx**

### CODE REVIEW

File: content/en/docs/appstore/connectors/aws-polly.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+## 3 Configuration
+
+After you install the connector, you can find it in the **App Explorer**, in the **AmazonPollyConnector** section. The connector provides a domain model and activities that you can use to connect your app to Amazon Polly. Each activity can be implemented using it in a microflow or nanoflow.
+
+For example, to synthesize a string of text into speech, implement the [SynthesizeSpeech](#synthesize-speech) activity by performing the following steps:
+
+1. In the **App Explorer**, right-click on the name of your module, and then click **Add microflow**.
+2. Enter a name for your microflow, for example, *ACT_SynthesizeSpeech*, and then click **OK**.
+3. In the **App Explorer**, in the **AmazonSNSConnector** section, find the **SynthesizeSpeech** activity.
```
Comment (brfreekmx) [inline content/en/docs/appstore/connectors/aws-polly.md]: Should be AmazonPollyConnector

File: content/en/docs/appstore/connectors/aws-polly.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+### 3.1 Constants
+
+Constants are used to define configuration values. All activities are exported as microflow activities that can directly be added to a microflow. Make sure the constants are configured correctly as shown in the table below, so the connector can authenticate the request with AWS.
+
+| Name | Description |
+| --- | --- |
+| `AmazonPollyConnector.AWS_ClientCertificateID` | The ID for the `ClientCertificate` used to sign the authentication requests. For more information, see [AWS Authentication](/appstore/connectors/aws-authentication/).
+ |
+| `AmazonPollyConnector.HostPattern` | The endpoint URL for the Amazon Polly service |
```
Comment (brfreekmx) [inline content/en/docs/appstore/connectors/aws-polly.md]: this line should be deleted

**Review by brfreekmx — CHANGES_REQUESTED** [review]
Can we remove the three screenshots as well? These are for the old app service and are no longer in line with the current implementation

File: content/en/docs/appstore/connectors/aws-polly.md (hunk 2 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws-polly.md]: Done.

File: content/en/docs/appstore/connectors/aws-polly.md (hunk 1 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws-polly.md]: Done.

File: content/en/docs/appstore/connectors/aws-polly.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
+---
+title: "Amazon Polly"
+url: /appstore/connectors/aws-polly/
```
Comment (brfreekmx) [inline content/en/docs/appstore/connectors/aws-polly.md:3]: can we make this amazon-polly

File: content/en/docs/appstore/connectors/aws-polly.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+category: "App Services"
```
Comment (brfreekmx) [inline content/en/docs/appstore/connectors/aws-polly.md:4]: Category is Connectors

File: content/en/docs/appstore/connectors/aws-polly.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+description: "Describes the configuration and usage of the Amazon Polly connector, which is available in the Mendix Marketplace."
+tags: ["marketplace", "marketplace component", "aws", "polly", "connector"]
```
Comment (brfreekmx) [inline content/en/docs/appstore/connectors/aws-polly.md:6]: Can we also add a tag amazon

File: content/en/docs/appstore/connectors/aws-polly.md (hunk 3 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws-polly.md:3]: Done.

File: content/en/docs/appstore/connectors/aws-polly.md (hunk 4 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws-polly.md:4]: Done.

File: content/en/docs/appstore/connectors/aws-polly.md (hunk 5 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/appstore/connectors/aws-polly.md:6]: Done.

**Pull request merged**